lib/tools/system-calls.js

```diff
@@ -411,6 +412,20 @@ systemCallMethods.launchAVD = async function (avdName, avdArgs, language, countr
   return proc;
 };
 
+systemCallMethods.getAdbVersion = async function () {
```

Comment:
I think it would be better to return the version number as an object, like

```
 {major: 1,
  minor: 1,
  build: 2}
```

in this case it will be easier to compare it.